README.md

```diff
@@ -93,19 +93,26 @@ An intf uses the primitives defined in pkgs to describe the interface between tw
 An intf consists of one or more components, and each component consists of one or more connections.  
 
 components are groups of individual port-to-port connections that constitute a higher-level protocol or logical grouping of signals.
-For example, if two blocks had a val/rdy handshake, the "val" port and the "rdy" port would each be described in a connection, and those connections would be contained within a component.
+Ports describe the shape of an interface (component) and connections describe the repeat of an interface (component).
```

Comment:
I understand what you're going for with 'shape' and 'repeat', but I think it would be clearer to say something like 'Ports describe the specific SV-types and widths that make up a component, and connections describe if the component is instantiated multiple times on the interface".

Reply:
I said similar below at each section's description. meant to avoid repeat of words that's why I looked for a "vague" word to replace.

Reply:
updated